### ISSUE TITLE
Questions in latest update (tinkers, other, dragons)

### ISSUE THREAD

**Issue by Vertigo81** [AllTheMods/atm6-sky#167]
A few things I can’t figure out.

tinkers book says it includes info on materials, but seems to be pretty much blank (just the page saying there is material info)

Unobtanium and vibranium need soul lava to be smelted in the tinkers smeltery?  But I can’t find a way to make/find it?  Is it in the other world gen somewhere or what am I missing?  The only source seems to be soul lava bees, which I haven’t the faintest clue how to get. (Or any of the resource bees)

are the variant slimes attainable?  Without the slime islands I’m not sure where they might be found.  Not the biggest deal but there are a few things that aren’t possible without them.

I reset my other dimension, and still can’t find any dragons.  I flew out 5000 blocks in a few directions and found nothing.  I did notice that the basilisks? No longer seem to be spawning, which is fine by me they we’re super annoying, but might be relevant?

I finally found a few fortresses in the other,  god they are annoying to find when buried.  I was looking for them in order to find Spawners to modify with apotheosis, but all the spawners I found are vanilla Minecraft, which I don’t think I can pick up or modify?  

I also still can’t figure out how to respawn the ender dragon :/  I’m sure I’m a total idiot and it’s something super simple, but I can’t seem to figure it out.  I need it for dragon breath and the quark dragon scale I think?

the only other way to get the breathe seems to be bees, which I can’t figure out either. *sigh*

**Comment by ryeenii** [comment]
> Unobtanium and vibranium need soul lava to be smelted in the tinkers smeltery? But I can’t find a way to make/find it? Is it in the other world gen somewhere or what am I missing? The only source seems to be soul lava bees, which I haven’t the faintest clue how to get. (Or any of the resource bees)

Resourceful bees are only gonna be craftable starting on 1.0.7, according to this comment by oly [here](https://github.com/AllTheMods/atm6-sky/issues/98#issuecomment-812980590).

**Comment by Eugene21040** [comment]
you can resurrect a dragon in a vanilla way(with end crystals) 
![image](https://user-images.githubusercontent.com/67188274/113600448-b5646780-9648-11eb-81ef-8be8b246f2a3.png)

and the soul lava in the old version of the assembly spawned in another world

**Comment by Topside08** [comment]
Apotheosis makes them collectable with silk touch. Forbidden and Arcanus has a craftable silk touch pick. Mekanism's Cardboard Box is probably the cheapest way to move them. I don't think they become Apotheosis spawners until you apply an upgrade or maybe when you mine them.

**Comment by Eugene21040** [comment]
> Apotheosis makes them collectable with silk touch. Forbidden and Arcanus has a craftable silk touch pick. Mekanism's Cardboard Box is probably the cheapest way to move them. I don't think they become Apotheosis spawners until you apply an upgrade or maybe when you mine them.


you can get a box of sawdust that is knocked out of a tree with a hammer. (method for early game)

**Comment by DankMemesys** [comment]
> tinkers book says it includes info on materials, but seems to be pretty much blank (just the page saying there is material info)

TiC is still in alpha, the in game documentation book is not complete, maybe a next TiC update will have the book with some info

**Comment by Vertigo81** [comment]
> TiC is still in alpha, the in game documentation book is not complete, maybe a next TiC update will have the book with some info

Thank you =)

> Apotheosis makes them collectable with silk touch. Forbidden and Arcanus has a craftable silk touch pick. Mekanism's Cardboard Box is probably the cheapest way to move them. I don't think they become Apotheosis spawners until you apply an upgrade or maybe when you mine them.

Ah thanks, I'll give it a try.  

> you can respawn a dragon in a vanilla way(with end crystals)
> ![image](https://user-images.githubusercontent.com/67188274/113600448-b5646780-9648-11eb-81ef-8be8b246f2a3.png)
> 
> and the soul lava in the old version of the assembly spawned in another world(allthemod)

I'll keep a lookout, haven't seen it though =/. I swear I've tried that exact thing (the vanilla spawning method).  There's no base in my end, just the central spike.  Maybe I have the height wrong or something? Or do they need to be placed on bedrock?  I used obsidian since there was nothing else there.

**Comment by Vertigo81** [comment]
Yeah I can't completely reproduce it, since the blocks that I would place the crystals on would be below the build limit.  My end has a 3 high pillar of bedrock only.  Standing on it you're at 0,3,0, the crystals should sit at 3,1,0, etc, but the block they should be placed on would be at 3,0,0, and can't be placed >_<

Going to try resetting my end.

**Comment by Vertigo81** [comment]
Ok that didn't work... The end reset, and now had pillars with end crystals (didn't the first time, was only a single spire in the void), but no dragon or portal frame or even the spire.  Just pillars in the void.  Not sure what else to try here >_<

**Comment by Eugene21040** [comment]
If there is a central block of the portal, can you complete the bedrock through creativity?

I read the problem about the lack of potal frames but did not see a solution

**Comment by Vertigo81** [comment]
> If there is a central block of the portal, can you complete the bedrock through creativity?
> 
> I read the problem about the lack of potal frames but did not see a solution

You mean can I rebuild the bedrock frame in creative?  I tried, since I was going to resort to file editing to try and make it work, figured why not.  The build limit prevented completing the existing structure, the bottom 2 layers would have been below the limit.  So I tried shifting it up by 2 blocks and it didn’t work.  I can place the crystals, but they don’t make their beams or spawn a dragon.

And now since resetting the end there’s no spire either.  Just the obsidian pillars and end crystals, iron cages etc.

**Comment by Vertigo81** [comment]
FYI I answered one of my own questions.  Skyslime is for sure obtainable.  Seems one of my farms must be located where a sky slime island would have been or something as I've had several large sky slimes spawn there.  Not sure about the others.

**Comment by Vertigo81** [comment]
Okay so after a LOT of research and struggle, I still can't fix it.  My end is just broken >_< very frustrating.

The portal being at below the build limit seems to break the ability to res pawn the ender dragon.  Without being able to respawn it a pile of things are unobtainable.  Dragon breath, the quark scale which means the elytra is a no go, which eliminates a lot of other flight related things.  I can cheat them in of course, but I hate doing it.

So, I reset the end hoping it would fix the issue.  But the end just respawns with pillars, no portal, and no dragon.  So still not spawnable.  I was confused, but after some research determined that the end dragon fight is separate from the end dimension.  And to reset the first kill I need to delete the NBT entry... So another rabbit hole.  

Turns out, there are no NBT editors that run on a Mac >_<. There used to be, but they aren't updated to run on more recent OS versions.  Tried to run it in emulation, but eventually gave up, luckily found an online editor.  Found the flag, deleted it, reset the end, went back in and YAY ender dragon.  Pillars, no portal.  So I grab some breath and kill the dragon.  It dies, spawns portal... In the exact same (under build limit) spot. >_<

#*&%#!  

So yeah.  I don't know what else to try.  Doing the end crystal dragon spawn seems to be completely impossible with my end, and I have no idea why it keeps doing this.

**Comment by Vertigo81** [comment]
FYI if anyone was still wondering about this.  I managed to solve it finally.

I learned that the portal location is actually saved in the level.dat, and isn't actually connected to the pillars or dragon or anything.  Mine was at 0,-1,0 which prevented the correct placement of the crystals for spawning.  I manually edited the .dat file and moved it up 10 blocks, set the dragon fight to false, then reset the End by deleting the dimension folder.  When I killed the dragon this time the portal spawned properly at the new location and im able to respawn it now.
